### PR TITLE
Add iPhone 12 specs

### PR DESCRIPTION
**Pull request by konan21**
Add new iPhone 12/mini/Pro/Pro Max specs and workaround for iPhone 12 mini. Also remove iPhone 4/4s from iOS 13 and 14.

### CODE REVIEW

**Comment by gajus** [comment]
> Also remove iPhone 4/4s from iOS 13 and 14.

What's the reason for removing it?

**Comment by konan21** [comment]
The reason is that iPhone 4 had last iOS 7.1.2 update and iPhone 4s - iOS 9.3.6

**Review by harshal-shinde — APPROVED** [review]

File: src/scream.js [hunk 1]
```diff
@@ -231,12 +233,22 @@ export default (config = {}) => {
...
         while (index--) {
-            if (window.screen.width === specs[index][4] &&
-                window.screen.height === specs[index][5] &&
-                window.devicePixelRatio === specs[index][6]) {
-                spec = specs[index];
-
-                break;
+            const currentSpec = specs[index];
+            if (window.screen.width === currentSpec[4] &&
+              window.screen.height === scurrentSpec[5] &&
```
Comment (harshal-shinde) [inline src/scream.js:238]: @konan21  typo for currentSpec

**Comment by balazsnemeth** [comment]
Any chance to have it rolled out to npm registry, @gajus ?

**Comment by gajus** [comment]
Done

**Pull request merged**